### DISCUSSION TITLE
Tasmota with Telegram an LD2410 motion detected notification

### DISCUSSION THREAD

**Discussion by murraa** [arendst/Tasmota#21903] [Q&A]
Rule1
	ON system#boot DO tmchatid XXXXXXXXX ENDON
	ON system#boot DO tmstate 1 ENDON
	ON system#boot DO tmtoken XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX ENDON
	ON system#boot DO tmsend LD2410 booted up! ENDON
	ON LD2410#Distance$![0.0,0.0,0.0] DO var1 %value%; DO backlog; tmsend LD2410: Motion Detected!; ENDON

I made this rule but I receive notifications when no movement is detected, where am I going wrong?

**Comment by sfromis** [comment]
Well, don't be creative with just inventing your own private syntax, not among what is documented for Tasmota. You can find that by looking for "array" after this point:
https://tasmota.github.io/docs/Rules/#examples-of-available-triggers
(Single array element only)

And only have *one* `DO` in a rule, and `Backlog` (if you even need that) at the start.
